app/src/main/java/de/westnordost/streetcomplete/ktx/Locale.kt

```diff
@@ -11,10 +11,6 @@ import java.util.Locale
  * @return String: The BCP 47 language tag for the current locale
  */
 fun Locale.toBcp47LanguageTag(): String? {
-    //if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
-    //    return toLanguageTag()
-    //}
-
     var language = language
```

Comment:
Heh, you might want to look at that again...

Reply:
My bad.